packages/get-platform/src/getPlatform.ts

```diff
@@ -255,7 +264,33 @@ export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetO
 }
 
 export async function getPlatform(): Promise<Platform> {
-  const { platform, libssl, distro, arch } = await getos()
+  const { platform, distro, arch, libssl } = await getos()
+
+  // sometimes we fail to detect the libssl version to use, so we default to 1.1.x
+  const defaultLibssl = '1.1.x' as const
+  if (platform === 'linux' && libssl === undefined) {
+    /**
+     * Ask the user to install openssl manually, and provide some additional instructions based on the detected Linux distro.
+     */
+    const additionalMessage = match({ distro })
+      .with({ distro: 'debian' }, () => {
+        return "Please manually install OpenSSL via `apt-get update -y && apt-get install -y openssl` and try installing Prisma again. If you're running Prisma on Docker, you may also try to replace your base image with `node:lts-slim`, which already ships with OpenSSL installed."
```

Comment:
Non-blocking, but something to consider.

Optimally we'd only tell the user to install the `libssl` library and not the whole OpenSSL distribution including the CLI which we don't need. However we don't know if `apt-get install -y libssl1.1` or `apt-get install -y libssl3` should be used on the current system. Do you have any ideas?

Maybe we could say something along the lines of "please run `apt-get install -y libssl1.1` or `apt-get install -y libssl3` depending on your system, or `apt-get install -y openssl` if you are not sure"?

Reply:
I'd keep the error message as simple as possible: users that care about minimal Docker images wouldn't use Debian in the first place imho. I think it's fine to ask them to install `openssl`, which automatically downloads a version of `libssl` compatible with the system, rather than us having to figure out whether they should opt for `libssl1.1` or other versions

Reply:
Optimally the error/warning message just says what the problem is (libssl could not be detected on the system) and how to fix it (install it) and then link to a docs page with more information (via pris.ly link), that explains the different approaches and so on.

But we can totally do that in a follow up.

Reply:
I'm totally fine with doing it in a follow-up PR, but I'd still prefer to have the consistent warnings with styling and functionality that the logger from `@prisma/internals` provides if possible